settings.gradle

```diff
@@ -0,0 +1,4 @@
+rootProject.name = 'terra-user-manager'
+include('service', 'client', 'integration', 'scripts')
```

Comment:
minor - Scripts does not have any source code. so may not be needed to be included in this file.